Mage/Task/BuiltIn/Deployment/Strategy/TarGzTask.php

```diff
@@ -132,7 +156,10 @@ public function run()
     protected function excludesListFile($excludesFile)
     {
         $excludesListFileRsync = '';
-        if (!empty($excludesFile) && file_exists($excludesFile) && is_file($excludesFile) && is_readable($excludesFile)) {
+        if (!empty($excludesFile)
+            && file_exists($excludesFile)
+            && is_file($excludesFile)
+            && is_readable($excludesFile)) {
```

Comment:
yes I know it was already there :) not your bad .. but I think it is to be cleaned up :)